ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
One suggestion. Its relatively little effort to automatically push images of different kinds. I suggest we tag images with :ubuntu-SHA, :alpine-SHA, :alpine-dbg-SHA with latest pointing to alpine/ubuntu or whatever.